h/views/admin/users.py

```diff
@@ -94,7 +94,7 @@ def users_rename(request):
     old_username = user.username
     new_username = request.params.get("new_username").strip()
 
-    try:
+    try:  # pylint:disable=too-many-try-statements
```

Comment:
I don't know, but I think the only statement required to be in the catch here is the `rename()`. Everything else looks like it could come before or after.

Reply:
I think you are right, moved the rest to a an `else`